src/modules/RA/DataLayer/components/TableConfigTab.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+import { BooleanInput, useTranslate } from 'react-admin';
+import { useField } from 'react-final-form';
+
+import Typography from '@material-ui/core/Typography';
+import Placeholder from '../../../../components/Placeholder';
+
+import TableConfigField from './TableConfigField';
+
+const validateTableFields = data => {
+  const valid = data.reduce((prev, field) => prev && (field.label.length > 0), true);
+  if (!valid) {
+    return 'datalayer.form.table.row-in-error';
+  }
+  return undefined;
+};
+
+const TableConfigTabContent = props => {
+  const { input: { value: tableEnable, onChange: onTableEnableChange } } = useField('table_enable');
+  const { input: { value: source } } = useField('source');
+  const { input: { value: fields } } = useField('fields');
+  const { input: { value: tableExportEnable } } = useField('table_export_enable');
+  const translate = useTranslate();
+
+  if (!source) {
+    return (<Placeholder><Typography variant="h5" component="h2">{translate('datalayer.form.table.no-source')}</Typography></Placeholder>);
```

Comment:
*(So long line should not pass linter validation)*

```suggestion
    return (
      <Placeholder>
        <Typography variant="h5" component="h2">
          {translate('datalayer.form.table.no-source')}
        </Typography>
      </Placeholder>
    );
```